src/markdown-pages/automate-workflows/diagnose-problems/error-alerts.mdx

```diff
@@ -213,4 +213,4 @@ At the end of your investigation, you discovered:
 
 Now, as a Telco Lite developer, you have enough information to debug the issue causing the memory leak. Congratulations!
 
-Learn more about using New Relic by diagnosing [other issues](diagnose-problems#view-your-services). If this is your last issue, [tear down](diagnose-problems#tear-down-telco-lite) all the Telco Lite services.
+Learn more about using New Relic by diagnosing [other issues](https://developer.newrelic.com/automate-workflows/diagnose-problems#view-your-services). If this is your last issue, [tear down](https://developer.newrelic.com/automate-workflows/diagnose-problems#tear-down-telco-lite) all the Telco Lite services.
```

Comment:
```suggestion
Learn more about using New Relic by diagnosing [other issues](/automate-workflows/diagnose-problems#view-your-services). If this is your last issue, [tear down](/automate-workflows/diagnose-problems#tear-down-telco-lite) all the Telco Lite services.
```

Good catch on these links being incorrect! Rather than using the full site URL, can you instead use links relative to the _root_? By doing it this way, we can maintain a nice SPA experience when navigating between these pages and avoid a page refresh.